frontend/src/app/api/tests/[testId]/questions/[questionId]/route.ts

```
@@ -0,0 +1,294 @@
+export async function GET(
+  request: Request,
+  { params }: { params: { testId: string; questionId: string } }
+) {
+  try {
+    const xata = getXataClient();
+    
+    // Validate IDs
+    const testId = parseInt(params.testId);
+    const questionId = parseInt(params.questionId);
+    
+    if (isNaN(testId) || testId <= 0 || isNaN(questionId) || questionId <= 0) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)